test/e2e/mcd_test.go

```diff
@@ -1122,3 +1125,218 @@ func setupForInternalImageRegistryPullSecretTest(t *testing.T, cs *framework.Cli
+
+	mode := 0644
+
+	mc := &mcfgv1.MachineConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "95-worker-install-aeskeyfind-x86-64",
+			Labels: map[string]string{
+				"machineconfiguration.openshift.io/role": "worker",
```

Comment:
issue: This MachineConfig gets rolled out to all of the worker nodes which can slow things down

suggestion: Create a MachineConfigPool just for this test, move the desired node into it, and apply the MachineConfig to just that node. You can use this helper to do all of that for you: hhttps://github.com/openshift/machine-config-operator/blob/master/test/helpers/utils.go#L135 and as a bonus, it has a cleanup function that will revert everything in the correct order.